<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnAddEmployee">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>280</y>
      <width>151</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Employee</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDisplayEmployees">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>320</y>
      <width>151</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Display Employees</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRemoveEmployee">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>360</y>
      <width>151</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove Employee</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnUpdateSalary">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>400</y>
      <width>151</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Update Salary</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSaveExit">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>440</y>
      <width>151</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Save &amp; Exit</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableEmployees">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>691</width>
      <height>221</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Full name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Age</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Sex</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Adress</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Passport</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Position code</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Salary</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Status</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
